zephyr/module.yml

```diff
@@ -2,3 +2,36 @@ name: hal_nordic
 build:
   cmake-ext: True
   kconfig-ext: True
+blobs:
+  - path: wifi_fw_bins/default/nrf70.bin
+    sha256: d321d489dfd2d34a75662d794c191f803ee0fe8afe742c3717142bdfa1959ecb
+    type: img
+    version: '1.0.0'
+    license-path: ./LICENSE.txt
+    url: https://github.com/nrfconnect/sdk-nrfxlib/raw/37e2ac22b6a6d9d9a62c95e8dc2c102d4371c62e/nrf_wifi/fw_bins/default/nrf70.bin
+    description: "nRF70 series firmware patch binary for default mode"
+    doc-url: https://github.com/nrfconnect/sdk-nrfxlib/tree/nrf_wifi/doc
```

Comment:
Invalid URL. I guess it was meant to be `https://github.com/nrfconnect/sdk-nrfxlib/tree/37e2ac22b6a6d9d9a62c95e8dc2c102d4371c62e/nrf_wifi/doc`.